### PR TITLE
Update ringing duration definition

### PR DESCRIPTION
**Pull request by davidlazarov**
set a maximum duration and the recommended duration

### CODE REVIEW

File: draft-detecting-unwanted-location-trackers.md [hunk 1]
```diff
@@ -493,7 +493,7 @@ Get_Identifier SHALL only be available when in identifier read state; otherwise,
 The identifier read state is discussed further in [Identifier Payload](#identifier-payload).
 
 ##### Play sound
-The Sound_Start opcode is used to play sound on the sound maker of the accessory. The sound maker MUST play sound for a minimum duration of 5 seconds.
+The Sound_Start opcode is used to play sound on the sound maker of the accessory. The sound maker MUST play sound for a minimum duration of 5 seconds and a maximum duration of 30 seconds. The recommended duration is 12 seconds.
```
Comment (bledvina) [inline draft-detecting-unwanted-location-trackers.md:497]: @davidlazarov should "recommended" be capitalized to conform to IETF requirement guidelines?

Comment (bledvina) [reply draft-detecting-unwanted-location-trackers.md:497]: @davidlazarov sorry, not capitalized, but all caps, i.e. "RECOMMENDED"

File: draft-detecting-unwanted-location-trackers.md [hunk 2]
```diff
@@ -493,7 +493,7 @@ Get_Identifier SHALL only be available when in identifier read state; otherwise,
...
```
Comment (bledvina) [inline draft-detecting-unwanted-location-trackers.md]: @davidlazarov should "recommended" be capitalized, i.e. "RECOMMENDED"

File: draft-detecting-unwanted-location-trackers.md (hunk 1 above)
Comment (davidlazarov) [reply draft-detecting-unwanted-location-trackers.md:497]: Done

File: draft-detecting-unwanted-location-trackers.md (hunk 2 above)
Comment (davidlazarov) [reply draft-detecting-unwanted-location-trackers.md]: Done

**Review by bledvina — APPROVED** [review]

**Pull request merged**